core/modules/locale/tests/locale.test

```diff
@@ -2808,3 +2808,46 @@ class LocaleLanguageNegotiationInfoFunctionalTest extends BackdropWebTestCase {
     $this->backdropGet('admin/config/regional/language/detection');
   }
 }
+
+/**
+ * Tests locale translation safe string handling.
+ */
+class LocaleStringIsSafeTest extends DrupalWebTestCase {
```

Comment:
Not sure, but I think that this needs to be `BackdropWebTestCase` instead (?). The only places we use `DrupalWebTestCase` seem to be in https://github.com/backdrop/backdrop/blob/1.x/core/includes/bootstrap.classes.inc#L130 and https://github.com/backdrop/backdrop/blob/1.x/core/includes/drupal.classes.inc#L41 and I believe that these are in place to provide wrapper/BC. Not an expert coder though, so take my words with a grain of salt.